astropy/time/tests/test_basic.py

```diff
@@ -906,16 +905,9 @@ def test_TimeFormat_scale():
     assert t.unix == t.utc.unix
 
 
+@remote_data
 def test_scale_conversion():
-    if INTERNET_OFF:
-        # With internet off (which is the default for testing) then this will
-        # fall back to the bundled IERS-B table and raise an exception.  But when
-        # testing with --remote-data the IERS_Auto class will get the latest IERS-A
-        # and this works.
-        with pytest.raises(ScaleValueError):
-            Time(Time.now().cxcsec, format='cxcsec', scale='ut1')
-    else:
-        Time(Time.now().cxcsec, format='cxcsec', scale='ut1')
+    Time(Time.now().cxcsec, format='cxcsec', scale='ut1')
```

Comment:
Does this mean we should then add a second test with @internet_off? (once that exists)

Reply:
I think so; that's why I opened this issue https://github.com/astropy/pytest-remotedata/issues/3. However, it's actually not clear to me how important it is to check for the exceptional condition here.

Reply:
@taldcroft - what would you recommend?

Reply:
Another consideration: would we then have to make sure to have at least one test run in the matrix with `--remote-data=none`?